fasthttpadaptor/adaptor.go

```diff
@@ -64,8 +64,8 @@ func NewFastHTTPHandler(h http.Handler) fasthttp.RequestHandler {
 				haveContentType = true
 			}
 
-			for _, v := range vv {
-				ctx.Response.Header.Add(k, v)
+			for i := range vv {
+				ctx.Response.Header.Add(k, vv[i])
```

Comment:
Same here.

Reply:
`[]string` is not the same as `[]*clientConn`, and I think the former produces a copy of the value, which may affect some performance in some extreme scenarios.

Reply:
can  you  give an example about  an extreme scenario？

Reply:
It may lead to excessive copying of values in some scenarios where many request headers need to be returned. Therefore, fetching by index can reduce the cost of copying this part of the value.

Reply:
But making a copy of a string is cheap as it's just 16 bytes. (pointer to data, length). It doesn't actually copy the value of the string. So I don't see that being an issue here.